plugins/abuseipdb/komand_abuseipdb/actions/check_ip/action.py

```diff
@@ -52,8 +52,8 @@ def run(self, params={}):
             )
 
         if len(out) > 0:
```

Comment:
```suggestion
         out[Output.ISMALICIOUS]  =  len(out) > 0
```
Is a cleaner way of writing the same thing.